src/Components/Rockets/Rockets.css

```diff
@@ -1,5 +1,29 @@
-* {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
+.container {
+  margin-top: 20px;
+}
+
+.flex {
+  display: flex;
+}
+
+.card {
+  gap: 15px;
+  margin: 25px 0;
+}
+
+.flickr {
+  width: 30%;
+}
+
+.flickr img {
+  width: 400px;
```

Comment:
I suggest that we use a more responsive unit for dimensions for example `width: 30%`